### PR TITLE
@ #563 | update window docs

### PR DESCRIPTION
**Pull request by datphan**
connect #563

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
...
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
 
+1. Install ``git``:
+
+Head over to https://git-scm.com/downloads and download their latest release then follow the instruction, by then you will have ``Git Bash`` already installed.
```
Comment (hoavt) [inline docs/getting_started.rst]: latest release then follow the instruction -> latest release, and follow the instruction to install git,

File: docs/getting_started.rst [hunk 2]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
...
+
+2. Install ``virtualbox``:
+
+Head over to https://www.virtualbox.org/ and download their latest release then follow the instruction, by then you will have ``virtualbox`` already installed.
```
Comment (hoavt) [inline docs/getting_started.rst]: their latest release, and follow the install instruction

**Comment by datphan** [comment]
@hoavt I've applied your comments, pls check again

File: docs/getting_started.rst [hunk 3]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
...
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
 
+1. Install ``git``:
+
+   Head over to https://git-scm.com/downloads and download their latest release, and follow the install instruction, by then you will have ``Git Bash`` already installed.
```
Comment (hoavt) [inline docs/getting_started.rst]: installation instruction

File: docs/getting_started.rst [hunk 4]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
...
+
+2. Install ``virtualbox``:
+
+   Head over to https://www.virtualbox.org/ and download their latest release, and follow the install instruction, by then you will have ``virtualbox`` already installed.
```
Comment (hoavt) [inline docs/getting_started.rst]: installation instruction

File: docs/getting_started.rst [hunk 5]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
...
+
+3. Install ``vagrant``:
+
+   Head over to https://www.vagrantup.com/ and download their latest release, and follow the install instruction, by then you will have ``vagrant`` already installed.
```
Comment (hoavt) [inline docs/getting_started.rst]: installation instruction

File: docs/getting_started.rst [hunk 6]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
 
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
 
+1. Install ``git``:
```
Comment (hoavt) [inline docs/getting_started.rst]: Install ``Git Bash``:

Comment (datphan) [reply docs/getting_started.rst]: no it is `git`, `Git Bash` is included in git

File: docs/getting_started.rst (hunk 5 above)
Comment (datphan) [reply docs/getting_started.rst]: applied

File: docs/getting_started.rst (hunk 4 above)
Comment (datphan) [reply docs/getting_started.rst]: applied

File: docs/getting_started.rst (hunk 3 above)
Comment (datphan) [reply docs/getting_started.rst]: applied

File: docs/getting_started.rst [hunk 7]
```diff
@@ -372,25 +390,35 @@ Git Setup
...
 
-          $ cat >> ~/.bash_profile
+Rsync with Git Bash
+-------------------
 
-      Type ``export VAGRANT_PREFER_SYSTEM_BIN=true`` and press ``Ctrl + D``, then run:
+To use rsync with `Git Bash`:
 
-       ..  code-block:: bash
+download [rsync-3.1](http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/rsync-3.1.2-2-x86_64.pkg.tar.xz) at http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/
```
Comment (hoavt) [inline docs/getting_started.rst]: Should have the bullet for this section:
- Download [rsync-3.1](http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/rsync-3.1.2-2-x86_64.pkg.tar.xz) at http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/

- Extract and copy `rsync.exe` to `C:\Program Files\Git\usr\bin`, then re-open your terminal window.

**Review by hoavt — COMMENTED** [review]
@datphan please check my comments.

File: docs/getting_started.rst (hunk 7 above)
Comment (datphan) [reply docs/getting_started.rst]: applied

**Review by hoavt — DISMISSED** [review]

File: docs/getting_started.rst [hunk 8]
```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
 
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
```
Comment (hoatle) [inline docs/getting_started.rst]: Please use `chocolatey` so that users can copy + paste commands to install required packages.

**Review by hoatle — DISMISSED** [review]
@datphan please check my comments.

File: docs/getting_started.rst (hunk 8 above)
Comment (datphan) [reply docs/getting_started.rst]: applied, pls check again

**Comment by hoatle** [comment]
assigned to @hoavt to continue reviewing.

File: docs/getting_started.rst [hunk 9]
```diff
@@ -150,17 +150,39 @@ Automatic Installation on Windows
...
-2. Install ``virtualbox``:
+   ..  code-block:: bash
 
-   Head over to https://www.virtualbox.org/ and download their latest release, and follow the installation instruction, by then you will have ``virtualbox`` already installed.
+       @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
 
-3. Install ``vagrant``:
+   After the installation is finished, restart the machine.
+
+2. Install ``git``:
```
Comment (hoavt) [inline docs/getting_started.rst]: should mention which versions of git, vagrant, virtualbox are accepted.

File: docs/getting_started.rst [hunk 10]
```diff
@@ -150,17 +150,39 @@ Automatic Installation on Windows
...
+  
+   Head over to https://git-scm.com/downloads then follow their installation instruction or use:
```
Comment (hoavt) [inline docs/getting_started.rst]: I think that instruction to install with the command below is enough for users.

Comment (hoavt) [inline docs/getting_started.rst]: Remove this line

**Review by hoavt — COMMENTED** [review]
@datphan please check my comments.

**Comment by datphan** [comment]
@hoavt applied your comments, pls check again

**Review by hoavt — DISMISSED** [review]

File: docs/getting_started.rst [hunk 11]
```diff
@@ -372,25 +406,35 @@ Git Setup
 To work with ``git``, complete the following guides to set up ssh keys:
 https://help.github.com/articles/connecting-to-github-with-ssh/
 
-   .. note::
+**Note:**
+
+- ~~On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash``~~.
```
Comment (hoatle) [inline docs/getting_started.rst]: move this to the cygwin section

File: docs/getting_started.rst [hunk 12]
```diff
@@ -372,25 +406,35 @@ Git Setup
...
+
+- ``Cygwin Terminal`` is now deprecated and no longer support, we are now using ``Git Bash``
```
Comment (hoatle) [inline docs/getting_started.rst]: not yet deprecated, we're supporting both at the moment. We'll decide to depreciate only when Git Bash is proven to be better and stable.

File: docs/getting_started.rst [hunk 13]
```diff
@@ -147,11 +146,46 @@ Automatic Installation on Windows
...
+4. Install [vagrant](https://www.vagrantup.com/) (>= 2.2.2):
+
+   ..  code-block:: bash
+
+       $ choco install vagrant --version 2.2.2
+
+Now everything is done, head over to [teracy-dev Git Clone and Vagrant Up](#teracy-dev-git-clone-and-vagrant-up) to continue the setup.
+
+.. _manual-installation-on-windows-deprecated:
+Manual Installation on Windows (**DEPRECATED**)
```
Comment (hoatle) [inline docs/getting_started.rst]: Manual Installation on Windows to use Cygwin

File: docs/getting_started.rst [hunk 14]
```diff
@@ -147,11 +146,46 @@ Automatic Installation on Windows
 
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
```
Comment (hoatle) [inline docs/getting_started.rst]: Manual Installation on Windows to use Git Bash

**Review by hoatle — COMMENTED** [review]
not yet to deprecate the cygwin stuff, we're adding git bash and must use it for a while to see if it's better. Until it's proven, we'll decide to depreciate then.

**Review by hoatle — CHANGES_REQUESTED** [review]
@datphan please check my comments.

File: docs/getting_started.rst (hunk 12 above)
Comment (datphan) [reply docs/getting_started.rst]: applied, pls check again, thanks

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**